Formula/jabba.rb

```diff
@@ -32,10 +32,21 @@ def install
   end
 
   test do
+    jdk_version = "openjdk@1.14.0"
+    version_check ='openjdk version "14'
+
+    on_macos do
+      if Hardware::CPU.arm?
+        jdk_version = "zulu@1.16.0-0"
+        version_check ='openjdk version "16'
+      end
+    end
```

Comment:
Let's just all install the same version
```suggestion
    jdk_version = "zulu@1.16.0-0"
    version_check ='openjdk version "16'
```